app/src/main/res/values/strings.xml

```diff
@@ -1200,6 +1200,12 @@
     <string name="preference_accessibility_force_enable_zoom">Zoom on all websites</string>
     <!-- Summary for Accessibility Force Enable Zoom Preference -->
     <string name="preference_accessibility_force_enable_zoom_summary">Enable to allow pinch and zoom, even on websites that prevent this gesture.</string>
+    <!-- Saved logins sorting strategy menu item -by name- (if selected, it will sort saved logins alphabetically) -->
+    <string name="saved_logins_sort_strategy_alphabetically">Name (A-Z)</string>
```

Comment:
I'm not sure if you'll know this, but do all languages have the concept of (A-Z)? If not, I'm assuming the translators will deal with this? Might be good to double check

Reply:
I used the string provided in the [mocks](https://app.abstract.com/share/33afd8de-f90e-4504-90e3-ee2a3bcb1925?mode=design&sha=0150fa57c67a9ae5786541d50682351760e767ca). Maybe we should ask @AmyYLee or @betsymi if the string `Name (A-Z)` is ok and the translators will deal with this afterwards.

Reply:
I ran this by l10n. We are already using this string in desktop, and communities have already adapted their translations. ([See Transvision](https://transvision.mozfr.org/string/?entity=browser/browser/aboutLogins.ftl:login-list-name-option&repo=gecko_strings)). What would need verification is if sorting and collation reflect the appropriate rules for the locale the user is using.